docs/wxm-devices/pulse/assemble.mdx

```diff
@@ -59,9 +59,9 @@ However, the wind vane’s movement has a small amount of friction, which assist
 
       Use a screwdriver to open the battery compartment and insert 2 AA batteries.
       Once the batteries are inserted, the LED indicator on the back of the sensor should turn on for 4 seconds
-      and then flash every 16 seconds indicating that the sensor transmits the data.
+      and then flash every 20 seconds indicating that the sensor transmits the data.
       Note that the LED flashing only indicates the transmission of the data from the outdoor sensor
-      and it does not assure the reception of the data by the WG3000 Gateway.
+      and it does not assure the reception of the data by the WG1200 Gateway.
```

Comment:
Wrong, it's WG3000 in Pulse. Please be careful of copy-pasting :)

Reply:
nice catch